lib/plugins/paragon-webpack-plugin/utils.js

```diff
@@ -0,0 +1,388 @@
+const { sources } = require('webpack');
+const parse5 = require('parse5');
+const axios = require('axios');
+
+async function getMfeRuntimeConfig() {
+  const {
+    APP_ID,
+    MFE_CONFIG_API_URL,
+  } = process.env;
+  const queryParams = new URLSearchParams({
+    mfe: APP_ID,
+  });
+  const url = `${MFE_CONFIG_API_URL}?${queryParams.toString()}`;
+  const response = await axios.get(url);
+  return response.data;
```

Comment:
Could we consider incorporating a try/catch block for this operation? It would be beneficial to anticipate and handle potential failures in the request.